Cargo.toml

```diff
@@ -471,13 +469,13 @@ libfalcon = { git = "https://github.com/oxidecomputer/falcon", branch = "main" }
 libnvme = { git = "https://github.com/oxidecomputer/libnvme", rev = "dd5bb221d327a1bc9287961718c3c10d6bd37da0" }
 linear-map = "1.2.0"
 live-tests-macros = { path = "live-tests/macros" }
-lldpd-client = { git = "https://github.com/oxidecomputer/lldp" }
+lldpd-client = { git = "https://github.com/oxidecomputer/lldp", branch = "bump_dendrite" }
```

Comment:
are you planning to change this branch, or is that the long term branch that continues to get used? Same for `dpd-client` above.

Reply:
The branch will go away.  This is just so I can test multiple in-flight repos at once.